packages/node/src/handlers/test-api.ts

```diff
@@ -0,0 +1,57 @@
+import find from 'lodash/find';
+import * as wallet from '../evm/wallet';
+import { randomString } from '../utils/string-utils';
+import { AggregatedApiCall, Config, WorkerOptions, ApiCallResponse } from '../types';
+import * as logger from '../logger';
+import { go } from '../utils/promise-utils';
+import { spawnNewApiCall } from '../adapters/http/worker';
+import { WORKER_CALL_API_TIMEOUT } from '../constants';
+
+export async function testApi(
+  config: Config,
+  endpointName: string,
+  parameters: Record<string, string>
+): Promise<[Error, null] | [null, ApiCallResponse]> {
+  const testCallId = randomString(8);
+  const masterHDNode = wallet.getMasterHDNode();
+
+  const logOptions = logger.buildBaseOptions(config, { requestId: testCallId });
+
+  const triggerRequest = find(config.triggers.request, ['endpointName', endpointName]);
+  if (!triggerRequest) {
+    return [new Error(`No such endpoint '${endpointName}'`), null];
+  }
+
+  const workerOpts: WorkerOptions = {
+    cloudProvider: config.nodeSettings.cloudProvider,
+    airnodeIdShort: wallet.getAirnodeIdShort(masterHDNode),
+    stage: config.nodeSettings.stage,
+    region: config.nodeSettings.region,
+  };
+
+  const aggregatedApiCall: AggregatedApiCall = {
```

Comment:
This feels weird to me but I don't have any better suggestions at the moment 😄

Reply:
Yeah, I agree. But using `spawnNewApiCall` felt like the best way to reuse node's code and I didn't want to change the types too much. So, "mocking" `aggregatedApiCall` seemed like a way to go :man_shrugging:

I can make those mocked fields optional and refactor the code to accomodate that if you think that would be better.

Reply:
> I can make those mocked fields optional and refactor the code to accomodate

Nah I don't think that's necessary. I like the strictness of making fields required